### PR TITLE
Fixed docker image deploy issues 

### PR DESCRIPTION
**Pull request by EmaNvl**

### CODE REVIEW

File: backend/docker-compose.yaml
Comment (March-08) [inline backend/docker-compose.yaml:1]: why docker compose is outside the containers folder? @EmaNvl

Comment (March-08) [reply backend/docker-compose.yaml:1]: Can you change back mlapp2 to mlapp? @EmaNvl

**Review by March-08 — CHANGES_REQUESTED** [review]
there are some things to fix

File: backend/docker-compose.yaml
Comment (EmaNvl) [reply backend/docker-compose.yaml:1]: - docker compose outside docker folder:
https://stackoverflow.com/questions/70117394/docker-cant-find-file-in-parent-folder#:~:text=had%20the%20same%20problem%20and%20I%20solved%20it%20by%20using%20docker%2Dcompose.

- you can fix the code? I don't know flusk so well

**Pull request merged**